utils/sl/fake_scons/SCons/Script/__init__.py

```diff
@@ -343,6 +343,9 @@ def Glob(*args):
     """Fake Glob"""
     return []
 
+def Split(*args):
```

Comment:
if you set this to _args instead of args, it will shut up the warning.  Obviously there will be others in this file but at least it doesn't add a new one

Reply:
fixed.